### PR TITLE
Support latest emoji on macOS

### PR DESCRIPTION
**Pull request by Dexwell**
## Type of change
Emoji such as 💇🏻‍♂️ no longer break up as the system emoji font is now used as a fallback. These emoji do not show up enlarged yet for some reason though.

## Screenshot
Before:
screen shot 2017-04-09 at 12 07 12

After:
screen shot 2017-04-09 at 12 53 55

### CODE REVIEW

File: app/style/common/variables.less [hunk 1]
```diff
@@ -18,11 +18,19 @@
  */
 
 // ----------------------------------------------------------------------------
+// Local Emoji Font
+// ----------------------------------------------------------------------------
+@font-face {
+					font-family: emoji;
+					src: local('Apple Color Emoji'), local('Android Emoji'), local('Segoe UI Emoji'), local('Segoe UI Symbol');
```
Comment (lipis) [inline app/style/common/variables.less]: let's have 2 spaces for indentation

Comment (Dexwell) [reply app/style/common/variables.less]: Changes pushed! I assume you meant two actual spaces; if you meant tabs let me know.

**Comment by lipis** [comment]
Thank you @Dexwell. Works nicely :)

**Comment by gregor** [comment]
Had to revert this in its current form as font style completely changed on Windows.

**Comment by Dexwell** [comment]
@gregor Sorry, I couldn't test on Windows. Could you check if changing

```css
src: local('Apple Color Emoji'), local('Android Emoji'), local('Segoe UI Emoji'), local('Segoe UI Symbol');
```

into just

```css
src: local('Apple Color Emoji');
```

resolves the issue?

**Comment by gregor** [comment]
@Dexwell Sure, will take a look.

**Pull request merged**